### DISCUSSION TITLE
Released dll

### DISCUSSION THREAD

**Discussion by T1GxR** [T1GxR/Axion-CS2-RAGE-CHEAT#3] [Show and tell]
This is the compiled dll for the cheat!
Use any injector to inject it using manual map.

<hr /><em>This discussion was created from the release <a href='https://github.com/T1GxR/Axion-CS2-RAGE-CHEAT/releases/tag/'>Released dll </a>.</em>

**Comment by ImHereTyt** [comment]
**_`It's a very interesting situation.`_**

If you compile a new source code, the size of the dll will be: 3,631 KB. And if you download the new DLL, the size is already 7,492 KB. The source breaks the game and doesn't start. But the released dll works). It's very interesting that you added something that you didn't insert into the source code?))

> Mb virus?)

![image](https://github.com/T1GxR/Axion-CS2-RAGE-CHEAT/assets/167651436/d491a963-2e3e-4a89-8df6-ae735c29188b)

**Comment by rfer3343** [comment]
sds

**Comment by WiwiBotic** [comment]
как и где обновить офсеты
